### PR TITLE
make zvmsdk log level debug to avoid too many logs

### PR DESCRIPTION
**Pull request by jichenjc**

### CODE REVIEW

File: zvmsdk/sdkwsgi/requestlog.py [hunk 1]
```diff
@@ -72,7 +72,10 @@ def _write_log(self, environ, req_uri, status, size, headers, exc_info):
...
-            LOG.info(self.format, log_format)
-        else:
-            LOG.debug(self.format, log_format)
+        # if LOG.isEnabledFor(logging.INFO):
+        #     LOG.info(self.format, log_format)
+        # else:
+        #     LOG.debug(self.format, log_format)
+
+        # Save space of log to avoid too large log file
+        LOG.debug(self.format, log_format)
```
Comment (SeanHQF) [inline zvmsdk/sdkwsgi/requestlog.py:81]: just curious if this will hide some necessary logs since the function is common?

**Review by bjhuangr — APPROVED** [review]

File: zvmsdk/sdkwsgi/requestlog.py (hunk 1 above)
Comment (jichenjc) [reply zvmsdk/sdkwsgi/requestlog.py:81]: yes, that's kind of trade off, discussed this with @bjhuangr on this 

we can enable debug if we need

Comment (haojue) [reply zvmsdk/sdkwsgi/requestlog.py:81]: Hi @jichenjc , is it possible that we just specifically disable these two logs mentioned in the issue to DEBUG level?then other logs will not be impacted?

Comment (jichenjc) [reply zvmsdk/sdkwsgi/requestlog.py:81]: yes, I thought this early but we decided that anyway we can enable the debug 
I will give a thought on this , thanks for the reminder

**Pull request merged**